app/assets/stylesheets/hosts.scss

```diff
@@ -36,3 +36,9 @@
     }
   }
 }
+
+#params {
+  legend {
```

Comment:
It was hard finding the correct size for the headers to distinguish them from the smaller headers over some of the tables.
I thought having it a bit bigger and the line underneath would make a clearer separation.